src/h5web/vis-packs/core/line/LineVis.tsx

```diff
@@ -38,7 +38,7 @@ function LineVis(props: Props) {
     ordinateLabel,
     title,
     errorsArray,
-    showErrors,
+    showErrors = false,
```

Comment:
Random change here, just to be a bit more explicit and consistent with `showGrid`.